publish/canisters/BUILD.bazel

```diff
@@ -69,8 +69,8 @@ CANISTERS_MAX_SIZE_COMPRESSED_E5_BYTES = {
     "ledger-canister_notify-method.wasm.gz": "9",
 
     # -- XC team --
-    # Size when constraint addded: 447_593 bytes
-    "ic-btc-kyt.wasm.gz": "6",
+    # Size when constraint addded: 649_790 bytes
```

Comment:
@THLO This size increase is somewhat surprising to me. Having `lazy_static` alone as dependency add roughly 50kB, so the other 150 kB seems to be due to using the Bitcoin address parsing...

Reply:
1. We ensure that every address is valid. Doing that, we for example realized that `TUCsTq7TofTCJRRoHk6RvhMoS2mJLm5Yzq` although in the OFAC checklist, is not a valid address 🤷‍♂️ .
2. We compare `Address` instances and not their textual representation when checking whether an address is blocked.

Reply:
Those are valid reasons, but I think such checks can be done as test cases. For example, we can add a test to ensure that the texture representation is canonical, like `assert_eq!(s, Address::try_from(s).unwrap().to_string())`.

Reply:
I agree that this would be possible and more lightweight alternative. Another for reason for this change was to try to unify a bit how the ethereum and bitcoin blocklist look like so that they can more easily be generated by @THLO 's scripts. Would the alternative suggested by Paul also ok for your scripts?

Reply:
Yes, the alternative would work, too. Given that there are some differences between the blocklist files for Bitcoin and Ethereum anyway, we can do what Paul suggested.

Reply:
@ninegua @THLO I reverted the changes regarding the Bitcoin blocklist and added a corresponding test. PTAL 🙏

Reply:
LGTM! Thanks a lot